MC/bin/o2dpg_sim_workflow.py

```diff
@@ -89,8 +89,17 @@
 workflow={}
 workflow['stages'] = []
 
+def relativeCPU(n_rel, n_workers=NWORKERS):
+    # compute number of CPUs from a given number of workers
+    # n_workers and a fraction n_rel
+    # catch cases where n_rel > 1
+    return min(n_workers, n_workers * n_rel)
```

Comment:
will this give a float (better) or an int?

Reply:
(actually we should probably cut the float at 2 decimals ... so that the JSON is still readable.

Reply:
Would a float `<1` also be okay? So say `0.15`?

Reply:
sure. It just means that the task uses around 15% of a full CPU.